src/Cli/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -100,7 +127,19 @@ public static int Run(ParseResult parseResult)
                 // one more time, there is no extra hop via msbuild
                 convertedArgs.AddRange(settings);
 
-                return new VSTestForwardingApp(convertedArgs).Execute();
+                if (FeatureFlag.Default.IsEnabled(FeatureFlag.ARTIFACTS_POSTPROCESSING))
```

Comment:
I am not sure why we need to add twice these arguments? Here and into msbuild args.

Reply:
For msbuild we pass argument how our MSBuild task understand. 
But we can also run 
```cs
dotnet test container.dll
```
In the case above the parameter has got different name than msbuild one...so sdk is splitted in two main branch, run through Msbuild and forward to vstest.console.dll directly(using dotnet exe vstest.console.dll --param).